### PR TITLE
return hex values in Method Parameters

### PR DESCRIPTION
**Pull request by ewilz**
fixes: https://github.com/Uniswap/universal-router-sdk/issues/92

### CODE REVIEW

File: src/utils/toHex.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
+// /**
+//  * Converts a big int to a hex string
+//  * @param bigintIsh
+//  * @returns The hex encoded calldata
```
Comment (marktoda) [inline src/utils/toHex.ts:4]: rm?

**Review by marktoda — COMMENTED** [review]
i think forge tests are gonna fail now no? should need some changes to parsing logic

File: src/utils/toHex.ts (hunk 1 above)
Comment (ewilz) [reply src/utils/toHex.ts]: you'rre too quick

**Review by marktoda — APPROVED** [review]
Lgtn

**Pull request merged**